buildscripts/kokoro/upload_artifacts.sh

```diff
@@ -33,6 +33,8 @@ LOCAL_OTHER_ARTIFACTS="$KOKORO_GFILE_DIR"/github/grpc-java/artifacts/
 
 # from macos job:
 [[ "$(find "$LOCAL_MVN_ARTIFACTS" -type f -iname 'protoc-gen-grpc-java-*-osx-x86_64.exe' | wc -l)" != '0' ]]
+# copy all x86 artifacts to aarch until native artifacts are built
+find "$LOCAL_MVN_ARTIFACTS" -type f -iname 'protoc-gen-grpc-java-*-osx-x86_64.exe*' -exec bash -c 'cp  ${0} ${0/x86/aarch}' {} \;
```

Comment:
Wrap the variables passed to cp in double quotes, just as general good shell hygiene.

nit: there's two spaces after cp.

Reply:
Cleaned up!